src/components/profile/edit/EditAccessGroups.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <EditMutationWrapper
+    :editVariables="{
+      accessInformationMozilliansorg: editMozilliansorgGroups.display,
+    }"
+    :initialValues="initialAccessInformation"
+    formName="Edit access groups"
+  >
+    <header class="profile__section-header" ref="header" tabindex="-1">
+      <h2>Access Groups</h2>
+      <PrivacySetting
+        class="privacy-setting--large"
+        label="Access groups privacy levels"
+        id="section-access-groups-privacy"
+        profileFieldName="accessInformation.mozilliansorg"
+        :profileFieldObject="editMozilliansorgGroups"
+        :collapsedShowLabel="true"
+      />
+    </header>
+    <IconBlockList
+      v-if="mozilliansorgGroups.length > 0"
+      class="icon-block-list--multi-col"
+    >
+      <IconBlock
+        v-for="[group] in mozilliansorgGroups"
+        :key="`group-${group}`"
+        icon="dino"
+      >
+        {{ group }}
+      </IconBlock>
+    </IconBlockList>
+    <p v-else>
+      You have not joined any mozilliansorg access group.
+    </p>
+    <a
+      id="access-information__external-link"
+      class="button button--secondary button--action"
+      href="https://mozillians.org"
```

Comment:
Have you considered linking to `https://mozillians.org/user/edit/#mygroups` to take a user directly to their groups settings in mozillians.org?

Reply:
I considered `https://mozillians.org/user/edit/` but I didn't realize that the groups there are actually links :sweat_smile: 
Thanks.